packages/core/elasticsearch/core-elasticsearch-client-server-internal/src/create_transport.ts

```diff
@@ -7,13 +7,13 @@
  */
 
 import type { IncomingHttpHeaders } from 'http';
-import type {
-  TransportRequestParams,
-  TransportRequestOptions,
-  TransportResult,
-} from '@elastic/transport';
-import type { TransportOptions } from '@elastic/transport/lib/Transport';
-import { Transport } from '@elastic/elasticsearch';
+import {
```

Comment:
Aside: are we going to standardize combining imports from the same place in this manner rather than declaring `type` imports separately?